src/console/controllers/ProjectConfigController.php

```diff
@@ -64,11 +83,82 @@ public function options($actionID): array
                 break;
             case 'diff':
                 $options[] = 'invert';
+                // no break
+            case 'get':
+                $options[] = 'fromFile';
+                // no break
```

Comment:
Why does `get` need `message`, `updateTimestamp`, and `force` options?

Reply:
Same as above